custom_components/ble_monitor/bt_helpers.py

```diff
@@ -148,7 +152,7 @@ def reset_bluetooth(hci):
     _LOGGER.debug("resetting Bluetooth")
 
     # Select the Bluetooth adapter and retreive the state of the adapter
-    mac = hci_get_mac([hci])[0]
+    mac = hci_get_mac([hci])[hci]
```

Comment:
Wasn't sure about this, but did a little test, you are right.

Reply:
Yeah, on my system i experimented with hci1, and the output of hci_get_mac was `{1: '00:1A:7D:DA:71:11'}`